ansible/eos_install_config

```diff
@@ -42,26 +45,39 @@ requirements:
 options:
     hostname:
         description: IP or FQDN of the device you want to connect to
-        required: true
+        required: True
     username:
         description: Username
-        required: true
+        required: True
     password:
         description: Password
-        required: true
+        required: True
     use_ssl:
```

Comment:
I thought about that for a brief moment yesterday, but I feel like the device libs (eos same as iosxr for that matter) are closer to the actual hardware and are ok to support vendor specific things, where napalm should unify them into one common interface (I would not include it there). But if someone uses pyEOS standalone and has https trouble - I find it ok to leave this option in for that use case.